helpers/deploymentConfig.ts

```diff
@@ -130,6 +130,10 @@ export const ADDRESSES: PreconfiguredAddresses = {
     WBTC: "0x92A2928f5634BEa89A195e7BeCF0f0FEEDAB885b",
     pufETH: "0x6D9f78b57AEeB0543a3c3B32Cc038bFB14a4bA68",
     LBTC: "0x37798CaB3Adde2F4064afBc1C7F9bbBc6A826375",
+    PTUSDe_27MAR2025: "0x74671106a04496199994787B6BcB064d08afbCCf",
+    USDe: "0x8bAe3E12870a002A0D4b6Eb0F0CBf91b29d9806F",
```

Comment:
Shouldn't we add the deployment file of the mock token for this asset (`USDe`) to https://github.com/VenusProtocol/isolated-pools/pull/457?